pages/list.vue

```diff
@@ -0,0 +1,212 @@
+            />
+          </div>
+        </header>
+
+        <ul class="divide-y divide-black/10">
+          <BookListItem
+            v-for="item in bookListStore.items"
+            :key="item.nftClassId"
+            :nft-class-id="item.nftClassId"
+            :price-index="item.priceIndex"
+            :is-selected="selectedItemIds.has(getBookListItemId(item.nftClassId, item.priceIndex))"
+            @click-cover="handleBookListItemCoverClick"
+            @remove="handleBookListItemRemove"
+            @select="handleItemSelect"
+            @unselect="handleItemDeselect"
+          />
+        </ul>
+      </div>
+      <div
+        v-else
+        class="flex flex-col items-center grow py-12"
+      >
+        <div class="flex flex-col justify-center items-center gap-2 py-4 grow text-muted">
+          <template v-if="bookListStore.isLoading">
+            <UIcon
+              class="animate-spin"
+              name="i-material-symbols-progress-activity"
+              size="48"
+            />
+            <span
+              class="font-semibold leading-none"
+              v-text="$t('book_list_loading_description')"
+            />
+          </template>
+          <template v-else>
+            <UIcon
+              name="i-material-symbols-favorite-outline-rounded"
+              size="48"
+            />
+            <span
+              class="font-semibold leading-none"
+              v-text="$t('book_list_empty_description')"
+            />
+          </template>
+        </div>
+        <UButton
+          :class="[
+            'max-w-[348px]',
+            'laptop:mt-12',
+            { 'opacity-0 pointer-events-none': bookListStore.isLoading },
+          ]"
+          leading-icon="i-material-symbols-storefront-outline"
+          :label="$t('book_list_empty_cta_button_label')"
+          size="xl"
+          block
+          :to="localeRoute({ name: 'store' })"
+        />
+      </div>
+    </div>
+  </main>
+</template>
+
+<script setup lang="ts">
+import { UCheckbox } from '#components'
+
+const { t: $t } = useI18n()
+const localeRoute = useLocaleRoute()
+const { loggedIn: hasLoggedIn, user } = useUserSession()
+const accountStore = useAccountStore()
+const bookListStore = useBookListStore()
+const { handleError } = useErrorHandler()
+const likeCoinSessionAPI = useLikeCoinSessionAPI()
+
+useHead({ title: $t('book_list_title') })
+
+const selectedItemIds = ref<Set<string>>(new Set())
+const hasSelectedItems = computed(() => selectedItemIds.value.size > 0)
+
+function handleSelectAllUpdate(isSelected: 'indeterminate' | boolean) {
+  if (isSelected) {
+    selectedItemIds.value = new Set(bookListStore.items.map(item => getBookListItemId(item.nftClassId, item.priceIndex)))
+  }
+  else {
+    selectedItemIds.value.clear()
+  }
+}
+
+function handleBookListItemCoverClick({ nftClassId, priceIndex }: BookListItem) {
+  useTrackEvent('view_item', { nftClassId, priceIndex })
```

Comment:
This seems duplicated. Cover click goes to `/store/:classId`, which already calls `view_item` on mount